test/unit_test/character_sheet_unit_tests/CharacterAttributeUnitTests.java

```diff
@@ -2,13 +2,50 @@
 
 import static org.junit.Assert.assertEquals;
 
+import java.security.InvalidParameterException;
+
+import org.junit.Rule;
 import org.junit.Test;
+import org.junit.rules.ExpectedException;
 
+import rpg_database.character_sheet.CharacterAttribute;
 import rpg_database.character_sheet.CharacterSheet;
 import rpg_database.character_sheet.Fields;
 
 public class CharacterAttributeUnitTests {
 
+	@Rule
+	public ExpectedException thrown = ExpectedException.none();
+
+	@Test
+	public void expectException_setSelfValueByFieldWrongObjectType() {
+		expectExceptionWithMessage(InvalidParameterException.class, "Unknown allowed field class: 'class java.lang.Integer'");
```

Comment:
Ennek az exceptionnek nincs így értelme, hiszen pont az a baj, hogy nem Integerrel állítjuk mégis azt az üzenetet küldi hogy ne Integerrel állítsuk! Itt az exception-t meg kell változtatni úgy (illetve az exception dobás mechanikáját is, ha nagyon aprólékos megoldást akarsz) hogy a valóságot tükrözze